### PR TITLE
Upgrade to regions 0.6

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR upgrades to regions 0.6

### CODE REVIEW

File: skyportal/tests/api/test_instrument.py [hunk 1]
```diff
@@ -346,9 +346,9 @@ def test_post_instrument_fov(super_admin_token):
...
     assert data['status'] == 'success'
-    print(data['data'])
 
-    assert (
-        data['data']['region']
-        == '# Region file format: DS9 astropy/regions\nfk5\ncircle(0.000006,0.000003,3.000000)\n'
-    )
+    region_str = """# Region file format: DS9 astropy/regions
+icrs
+circle(0.00000000,0.00000000,3.00000000)"""
```
Comment (guynir42) [inline skyportal/tests/api/test_instrument.py:352]: what happened to the indentation here?

File: skyportal/tests/api/test_instrument.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def test_token_user_post_get_instrument(super_admin_token):
         [
             d['field_id'] == 1
             and d['contour']['features'][0]['geometry']['coordinates'][0][0]
-            == [110.84784299030288, -87.01522509948724]
+            == [110.84791974982103, -87.01522999646508]
```
Comment (guynir42) [inline skyportal/tests/api/test_instrument.py:89]: I'm curious why the coordinates changed?

**Review by guynir42 — APPROVED** [review]

File: skyportal/tests/api/test_instrument.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/tests/api/test_instrument.py:89]: I think some subtle things in the round trip.

**Comment by mcoughlin** [comment]
Thanks @guynir42

**Pull request merged**